stock_procurement_customer/models/stock_picking.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    customer_id = fields.Many2one(
```

Comment:
@rousseldenis This field should be displayed on the picking view if the value is filled and different of the partner_id.